spec/index.html

```diff
@@ -360,12 +363,26 @@ <h3>Triple Terms and Reification</h3>
           aria-label="An RDF graph containing a reifying triple where the triple term corresponds to an asserted triple"></object>
       </a>
       <figcaption id="fig-asserted-triple-term-alt">
-        An <a>RDF graph</a> containing a <a>reifying triple</a> where the <a>triple term</a> corresponds to an <a>asserted triple</a>.
+        An <a>RDF graph</a> containing a <a>triple annotation</a>, which is a <a>reifying triple</a> whose <a>triple term</a> corresponds to an <a>asserted triple</a>.
         The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
       </figcaption>
     </figure>
 
-    <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
+    <p class="note">
+      A <a>proposition</a> denoted by a <a>triple term</a> is an abstract,
+      logical relationship; assertions are always made about some
+      <a>reifier</a> thereof. There can be multiple, distinct reifiers
+      underlying the same abstract proposition, such as statements with
+      different sources, or circumstances with different characteristics. Since
+      a proposition so reified does not have to hold, it is possible to make
```

Comment:
Per the suggestion by @franconi, slightly rephrased:
```suggestion
      different sources, or circumstances with different characteristics.
      One reifier may also reify multiple, distinct propositions, expressing
      different propositional aspects of the same reifier.
      Since a proposition so reified does not have to hold, it is possible to make
```
While I strongly agree with this, in case there is some concern about this as common practice, I'd accept not spelling it out (cf. the possible removal regarding nested triples below). It is still a possible practice of course.

Reply:
I'm inclined to add this. (I updated the suggestion to "use" of reifier, to avoid conflating with its referent.)

Reply:
Please proceed!

Reply:
Yes; no adverse opinions and lots of eyes here; so committing this. There's still some other suggested adjustments, so I'll suggest leave the PR open for any final reactions. (As I'm not editor here, I won't merge myself, or at least not without an editor asking me to do so.)